docs/_quickstart/introduction.md

```diff
@@ -9,17 +9,19 @@ redirect_from: /quick-start/
 
 [Zapier](https://zapier.com/){:target="_blank"} is an app automation platform where over 2 million people connect apps into customized workflows—what we call Zaps. A Zapier integration can connect your app's API to the Zapier platform and let it integrate with {{ site.partner_count }} other popular apps to watch for new or updated data, find existing data, or create and update data.
 
-![Example Zap](https://cdn.zapier.com/storage/photos/bcad0a485f61f4acccf36cb19a0261ec.gif)
+![Example Zap](https://cdn.zappy.app/eeff50e2737492689f688c379c1db65a.gif)
 
 _Building an example Google Forms to GitHub Zap in Zapier's editor._
 
 For example, say you want to turn bugs submitted in a Google Form into GitHub issues. A Zap like the one above could watch your Google Form for new entries, then use the form data to make a new GitHub issue. Fill out the form anytime to add a new issue.
 
-A Zapier integration for your app lets your users build workflows like that and more with your app.
+A Zapier integration for your app lets your users build workflows like that and more with your app. Check out a demo [here](https://zapier.com/demo). 
 
-Zaps are built of steps—or functions that take input, communicate with an API, and return output. Users build Zaps from two or more steps from integrations. The first step is always the trigger, that watches for new or updated data every few minutes (with polling triggers) or when your APIs tells us to (with subscription triggers), and starts the Zap. Subsequent steps—or actions—then use that data to find or add items to an app, with search actions to find data with GET calls or create actions to add new items with PUT or POST calls.
+Each Zap created starts with a trigger (something that happens in one app) and then one or more actions (something else that happens in another app). These triggers and actions take input, communicate with an API, and return output. 
 
-![Zapier Fields](https://cdn.zapier.com/storage/photos/e0442350236db38688da231caafdab5f.gif)
+The first step is always the trigger, that watches for new or updated data every few minutes (with polling triggers) or when your APIs tells us to (with subscription triggers), and starts the Zap. Subsequent steps—or actions—then use that data to find or add items to an app, with search actions to find data with GET calls or create actions to add new or update existing items with POST or PUT calls.
```

Comment:
Small typo: `your APIs tells`

Should be either `your API tells` or `your APIs tell`